tests/e2e/ctl_v3_auth_test.go

```diff
@@ -1265,3 +1265,24 @@ func ctlV3EndpointHealth(cx ctlCtx) error {
 	}
 	return e2e.SpawnWithExpects(cmdArgs, cx.envMap, lines...)
 }
+
+func ctlV3User(cx ctlCtx, args []string, expStr string, stdIn []string) error {
```

Comment:
Please add a comment here something like:  `This function may need to be deleted when the tests in ctl_v3_auth_test.go are migrated. `

Reply:
I think this can be said about every function, `delete it when it's not used` :P

Reply:
Yeah I'm not sure a comment makes sense - anyone migrating the tests will presumably also delete any code they can. (And that person is probably going to be next week me)

Reply:
It is OK because the function `ctlV3User` is in the same file as the test cases.